src/main/java/com/wanderersoftherift/wotr/init/WotrItems.java

```diff
@@ -107,6 +109,65 @@ public class WotrItems {
+    public static final DeferredItem<Item> HONEY_ESSENCE = registerEssence(
+            EssenceType.HONEY.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> MUSHROOM_ESSENCE = registerEssence(
+            EssenceType.MUSHROOM.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> FABRIC_ESSENCE = registerEssence(
+            EssenceType.FABRIC.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> DARK_ESSENCE = registerEssence(
+            EssenceType.DARK.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> FIRE_ESSENCE = registerEssence(
+            EssenceType.FIRE.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> AIR_ESSENCE = registerEssence(
+            EssenceType.AIR.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> ENERGY_ESSENCE = registerEssence(
+            EssenceType.ENERGY.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> ANIMAL_ESSENCE = registerEssence(
+            EssenceType.ANIMAL.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> CRYSTAL_ESSENCE = registerEssence(
+            EssenceType.CRYSTAL.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> METAL_ESSENCE = registerEssence(
+            EssenceType.METAL.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> FOOD_ESSENCE = registerEssence(
+            EssenceType.FOOD.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> SLIME_ESSENCE = registerEssence(
+            EssenceType.SLIME.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> MIND_ESSENCE = registerEssence(
+            EssenceType.MIND.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> MECHA_ESSENCE = registerEssence(
+            EssenceType.MECHA.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> END_ESSENCE = registerEssence(
+            EssenceType.END.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> FLOW_ESSENCE = registerEssence(
+            EssenceType.FLOW.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> FORM_ESSENCE = registerEssence(
+            EssenceType.FORM.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> ORDER_ESSENCE = registerEssence(
+            EssenceType.ORDER.getName().toLowerCase() + "_essence");
+    public static final DeferredItem<Item> CHAOS_ESSENCE = registerEssence(
+            EssenceType.CHAOS.getName().toLowerCase() + "_essence");
+
+    public static final DeferredItem<Item> registerEssence(String essenceName) {
+        return ITEMS.register(essenceName, registryName -> new EssenceItem(
```

Comment:
```suggestion
        return ITEMS.register(essenceName, registryName -> new Item(
```
Because the `EssenceItem` class is not needed, this can be changed to just be `Item`